vsg/tests/comment/test_rule_comment.py

```diff
@@ -27,7 +27,7 @@ def test_rule_010(self):
 
     def test_rule_010_library(self):
         oRuleList = rule_list.rule_list(oFileLibrary)
-        oRuleList.check_rules()
+        oRuleList.check_rules([])
```

Comment:
Maybe there should be default value for this argument, there will be no need to pass empty table `[]`.

Reply:
That makes perfect sense.  I will update.

Reply:
I just pushed an update.